lib/RTCSession.js

```diff
@@ -1320,7 +1320,9 @@ module.exports = class RTCSession extends EventEmitter
 
     const originalTarget = target;
 
-    if (this._status !== C.STATUS_WAITING_FOR_ACK && this._status !== C.STATUS_CONFIRMED)
+    if (this._status !== C.STATUS_WAITING_FOR_ACK 
+      && this._status !== C.STATUS_CONFIRMED 
+      && this._status !== C.STATUS_1XX_RECEIVED)
```

Comment:
I do not understand this. Why shouldn't the UA be able to refer the remote after receiving a 180 or 183?

Reply:
because jssip allow refer only after receiving 200 OK, if status was STATUS_1XX_RECEIVED ( e.g. 180 Ringing) the next piece of code was `return false;` and this block all refer procedure.

I think this was made because current code does not create RTCSession._dialog when receiving 180

Reply:
Let's please wait for @jmillan to come back to activity in 2-3 weeks, he is the mind in this piece of code.

Reply:
I've commented here: https://github.com/versatica/JsSIP/issues/566